### ISSUE TITLE
Add eo-runtime dependency for all eo programs

### ISSUE THREAD

**Issue by volodya-lombrozo** [objectionary/eo#1361]
Issue comes from https://github.com/objectionary/eoc/issues/35 

I’ve tried to run the same code in `sandbox` and using `eoc` directly:

```eo
[a b] > sum
  plus. > @
    a
    b
```
In both cases we have the same compilation exception.
The problem remains with all EO programs where we don’t use foreign objects. For example, the next code samples will compile fine:
```eo
[a b] > sum
  plus. > @
    a
    b
  QQ.io.stdout > neo
    "Wake up, Neo..."
```

Or

```eo
[a b] > sum
  memory 0 > x
  plus. > @
    a
    b
```

As I mentioned above, in the first code sample we don’t use foreign objects, by that reason our `eo-foreign.json` will contain only one entry:
```json
[
    {
        "id": "sum",
        "discovered": "0",
        "xmir": "/Users/lombrozo/Workspace/EOlang/Projects/sum/.eoc/01-parse/sum.xmir",
        "scope": "compile",
        "eo": "/Users/lombrozo/Workspace/EOlang/Projects/sum/sum.eo",
        "xmir2": "/Users/lombrozo/Workspace/EOlang/Projects/sum/.eoc/03-optimize/sum.xmir",
        "version": "0.0.0"
    }
]
``` 

Therefore the next phases `Discover`,  `Pull`, `Resolve`, `Place` will do nothing.  The key point there is that classpath after all phases will be empty. `eo-runtime` classes will absent and we will see compilation error (since all transpiled classes depend on runtime library).

**Comment by 0pdd** [comment]
@volodya-lombrozo 2 puzzles [#1385](https://github.com/objectionary/eo/issues/1385), [#1386](https://github.com/objectionary/eo/issues/1386) are still not solved.

**Comment by 0pdd** [comment]
@volodya-lombrozo the puzzle [#1385](https://github.com/objectionary/eo/issues/1385) is still not solved; solved: [#1386](https://github.com/objectionary/eo/issues/1386).

**Comment by 0pdd** [comment]
@volodya-lombrozo all 2 puzzles are solved here: [#1385](https://github.com/objectionary/eo/issues/1385), [#1386](https://github.com/objectionary/eo/issues/1386).
